src/core/services/error-handling-service.ts

```
@@ -25,7 +28,7 @@ export class ErrorHandlingService {
       }
 
       return c.json(
-        this.createResponse("FATAL_ERROR", "Internal server error"),
+        this.createResponse("FATAL_ERROR", INTERNAL_SERVER_ERROR_MESSAGE),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)